content/developer/reference/backend/orm.rst

```diff
@@ -859,6 +859,8 @@ Search/Read
 
 .. automethod:: Model.read
 
+.. automethod:: Model._read_group
```

Comment:
See https://github.com/odoo/odoo/pull/110737#pullrequestreview-1391805363 for the review of the docstring.